src/e2e/kotlin/uk/gov/justice/digital/hmpps/deliusapi/EndToEndTest.kt

```diff
@@ -85,31 +86,33 @@ abstract class EndToEndTest {
   }
 
   private fun getToken(): String {
-    if (CACHED_TOKEN?.expired() == false) {
-      return CACHED_TOKEN!!.accessToken
+    if (CACHED_TOKEN?.expired() != false) {
```

Comment:
Can you remove the condition completely here?

Reply:
I don't think so, it's effectively saying - if the CACHED_TOKEN is `null`, or CACHED_TOKEN.expired() is `true`, then fetch the token.  It is confusing though, maybe being more explicit is better e.g.
```suggestion
    if (CACHED_TOKEN == null || CACHED_TOKEN.expired()) {
```

Reply:
OK yes - that is clearer